multi_step_wizard/views/multi_step_wizard_views.xml

```diff
@@ -1,23 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-  <record id="multi_step_wizard_form" model="ir.ui.view">
-    <field name="name">multi.step.wizard.form</field>
-    <field name="model">multi.step.wizard.mixin</field>
-    <field name="arch" type="xml">
-      <form>
-        <field name="state" invisible="1"/>
-        <footer>
-          <div name="states_buttons" attrs="{'invisible': [('state', '=', 'final')]}">
-            <button name="open_next" string="Next" type="object" class="btn-primary"/>
-            <button string="Cancel" class="btn btn-default" special="cancel" />
-          </div>
-          <div name="final_buttons" attrs="{'invisible': [('state', '!=', 'final')]}">
```

Comment:
Were did the Close button go?

Reply:
It seems to be an issue in v11 PR as well

Reply:
Well, it's the cancel button in line 17 of the new version

Reply:
True, sorry :)